open-metadata-test/open-metadata-fvt/open-types-fvt/open-types-test/pom.xml

```diff
@@ -42,7 +42,7 @@
         </dependency>
         <dependency>
                <groupId>org.odpi.egeria</groupId>
-               <artifactId>repository-services-implementation</artifactId>
+               <artifactId>open-metadata-types</artifactId>
                <version>1.1-SNAPSHOT</version>
```

Comment:
technically this version shouldn't be here (though it wasn't a change in this PR). I will do a search/scan to fix up any hanging over or re-introduced and slipped through the net